tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-initial-dialog.html

```diff
@@ -29,17 +29,13 @@ <h2 id='dialog-title'>Waiting for first Session.run() to connect...</h2>
 # To connect to the debugger from your tf.Session:
 from tensorflow.python import debug as tf_debug
 sess = tf.Session()
-def watch_fn(fetches, feeds):
-  return tf_debug.WatchOptions(debug_ops=["DebugIdentity(gated_grpc=true)"])
-sess = tf_debug.GrpcDebugWrapperSession(sess, "[[_host]]:[[_port]]", watch_fn=watch_fn)
+sess = tf_debug.TensorBoardDebugWrapperSession(sess, "[[_host]]:[[_port]]")
 sess.run(my_fetches)
         </pre>
         <pre id="hook-code-example">
 # To connect to the debugger using hooks, e.g., from tf.Estimator:
 from tensorflow.python import debug as tf_debug
-def watch_fn(fetches, feeds):
-  return tf_debug.WatchOptions(debug_ops=["DebugIdentity(gated_grpc=true)"])
-hook = tf_debug.GrpcDebugHook("[[_host]]:[[_port]]", watch_fn=watch_fn)
+hook = tf_debug.TensorBoardDebugHook("[[_host]]:[[_port]]")
```

Comment:
Maybe note that the host could be prefixed with `grpc://` or not. Either way works.

Reply:
Since both way works, let's just mention the simpler way here. To avoid unnecessary  decision paralysis. :)